osu.Game/Screens/Edit/Compose/Components/Timeline/Timeline.cs

```diff
@@ -57,7 +62,9 @@ private void load(IBindable<WorkingBeatmap> beatmap, OsuColour colours)
             Beatmap.BindValueChanged(b =>
             {
                 waveform.Waveform = b.NewValue.Waveform;
-                track = b.NewValue.Track;
+                track = musicController.CurrentTrack;
+
+                Debug.Assert(track != null);
```

Comment:
This assert feels a little strange seeing that `musicController.CurrentTrack` is already annotated as `[NotNull]`. I know it's a jetbrains annotation and therefore non-enforceable on a compiler level (although I *think* we have inspections for that enabled), but why guard this place but not the others?